ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -770,6 +770,30 @@ FCU_DFMSS_DSP_ZHM_RTM_CHWZC_CHWDT:
   - CHWZC
   - CHWDT
 
+FCU_DFSS_STM_RTC_RHM:
+  description: "CRAC unit cooling and humidification with compressor and alarm monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - DFSS
+  - STM
+  - RTC
```

Comment:
dx? chw? need to clarify, even if the unit doesnt provide complete data for the type. use missing fields instead.

Reply:
Used DXRC